src/test/java/gomedic/logic/parser/DateTimeParserTest.java

```diff
@@ -0,0 +1,5 @@
+package gomedic.logic.parser;
+
+public class DateTimeParserTest {
```

Comment:
Actually, maybe you can try to write some test cases just use assertTrue(isValidDateTime) etc just to check that it works correctly